plugins/GSdx/Window/GSWndEGL.cpp

```diff
@@ -205,10 +208,10 @@ bool GSWndEGL::Create(const std::string& title, int w, int h)
 
 	m_managed = true;
 
+	m_native_display = CreateNativeDisplay();
 	OpenEGLDisplay();
 
 	m_native_window = CreateNativeWindow(w, h);
-
```

Comment:
same here

Reply:
reasoning is dependencies, you need to set `m_native_display` before you can call `OpenEGLDisplay`, then you need to set `m_native_window` before you can call `FullContextInit`.

In the Attach case the `AttachNativeWindow` assigns both of those variables so I squished all three calls next to each other to signify that they're closely related and not separate steps I guess.

Let me know if you still prefer the newlines.

Reply:
Fine by me